src/v0/destinations/ga4/transform.js

```diff
@@ -84,7 +96,7 @@ const responseBuilder = (message, { Config }) => {
     case 'gtag':
       // gtag.js uses client_id
       // GA4 uses it as an identifier to distinguish site visitors.
-      rawPayload.client_id = getGA4ClientId(message);
+      rawPayload.client_id = getGA4ClientId(message, Config);
```

Comment:
is overriding of client id happening only hybrid? for device more we don't want to override right?